src/Database/Migrations/2021-07-04-041948_CreateSettingsTable.php

```diff
@@ -2,10 +2,22 @@
 
 namespace CodeIgniter\Settings\Database\Migrations;
 
+use CodeIgniter\Config\BaseConfig;
+use CodeIgniter\Database\Forge;
 use CodeIgniter\Database\Migration;
 
 class CreateSettingsTable extends Migration
 {
+    private BaseConfig $config;
+
+    public function __construct(?Forge $forge = null)
+    {
+        $this->config  = config('Settings');
+        $this->DBGroup = (isset($this->config->database['group']) && $this->config->database['group']) ? $this->config->database['group'] : null;
```

Comment:
Also okay to assume the required database key:
```suggestion
        $this->DBGroup = $this->config->database['group'];
```

Reply:
Actually now that I think of it you don't use the `$Config` property at all. Please drop that and make this a simple assignment:
```
        $this->DBGroup = config('Settings')->database['group'];
```